src/components/Modals/AddEditForm/addeditform.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import FormInput from '../../FormInput/forminput';
+
+const AddEditForm = ({ formik, type }) => {
+  return (
+    <>
+      <FormInput title="TITLE" id="title" name="title" type="text" onChange={formik.handleChange} value={formik.values.title} onBlur={formik.handleBlur} />
+      {formik.touched.title && formik.errors.title ? (
+        <div>{formik.errors.title}</div>
+      ) : null}
+      <FormInput title="RELEASE DATE" id="release_date" name="release_date" type="date" onChange={formik.handleChange} value={formik.values.release_date} onBlur={formik.handleBlur} />
+      {formik.touched.release_date && formik.errors.release_date ? (
+        <div>{formik.errors.release_date}</div>
+      ) : null}
```

Comment:
Let's get error handling into the component as well

Reply:
done